tools/c7n_oci/c7n_oci/provider.py

```diff
@@ -21,13 +47,68 @@ class OCI(Provider):
     resource_map = ResourceMap
 
     def initialize(self, options):
+        if options["account_id"] is None:
+            options["oci_config"] = get_oci_profile_session(options).get_config()
         return options
 
+    def _get_subscribed_region(self, options):
+        client = get_oci_profile_session(options).client("oci.identity.IdentityClient")
+        return client.list_region_subscriptions(tenancy_id=options["oci_config"]["tenancy"]).data
+
+    def _is_subscribed_region(self, subscribed_regions, region, policy_collection):
+        for r in subscribed_regions:
+            if r.region_name == region and r.status == "READY":
+                return True
+
+        policy_collection.log.info(
```

Comment:
would suggest using a dedicated logger for this (c7n.oci) at module scope. policy collection is generic and not scoped to oci provider and the message is specific to that.

Reply:
Done.